doc/examples/plot_register_translation.py

```diff
@@ -83,3 +83,40 @@
 
 print("Detected subpixel offset (y, x):")
 print(shift)
+
+#  Detecting the shift and using fourier_shift from scipy to subpixel shifting
+#    the image such that the two overlap. This is done via a FFT and and iFFT
+# This is useful for overlapping two or more images. Use fftn/ifftn instead of
+#    fft2/ifft2 for RGB images
+
+offset_image = offset_image.real  # A normal image isn't complex
+shift, error, diffphase = register_translation(image, offset_image, 100)
+print("Detected subpixel offset (y, x):")
+print(shift)
+shifted_image = np.fft.ifft2(
+    fourier_shift(np.fft.fft2(offset_image), shift)).real
+
+# Round the image to integers and convert to correct dtype
+shifted_image = np.around(shifted_image).astype(image.dtype)
+
+fig, (ax1, ax2, ax3) = plt.subplots(ncols=3, figsize=(8, 3))
+ax1.imshow(offset_image)
+ax1.set_axis_off()
+ax1.set_title('Offset image')
+
+ax2.imshow(shifted_image)
+ax2.set_axis_off()
+ax2.set_title('Shifted back image')
+
+
+diff = image - shifted_image
```

Comment:
You should convert to float before taking the difference, using ``img_as_float``, otherwise overflows can happen.